record-files/record_assignments.csv

```diff
@@ -0,0 +1,163 @@
+Record,Users Assigned

```

Comment:
Maybe we shouldn't add assigned users yet, just leave it blank when it gets pushed to the server then update it there

Reply:
Yeah that would make sense... I forgot to clear it after testing when I pushed to GitHub so that's my mistake!